src/Appwrite/Resque/Worker.php

```diff
@@ -14,16 +21,138 @@ abstract public function shutdown(): void;
 
     public function setUp(): void
     {
```

Comment:
I think we should have here an error handler callback that we can register on the init file. This implementation couples the Worker and Logger libs in a way we can avoid. 

I think we can try to have a similar implementation to what we have in in utopia/fw, where you pass an error callback and the library triggers it in correct places. This will also allow us to dramatically reduce the code duplication and complexity the error handler has added to this class. 

@Meldiron I would love to chat about this specific pattern later, let me know when you have time.